policies/release-policy.md

```diff
@@ -0,0 +1,123 @@
+available time and is not prioritised higher than any existing task within the
+release backlog.
+
+This phase concludes with an alpha release readiness check by the steering
+committee. Upon successful completion of this check, an alpha release is
+created, marking the beginning of the Alpha phase.
+
+### 4. Release
+The Release phase of OpenSSL library commences with the creation of an alpha
+release at the end of the Development phase and spans over a period of six
+weeks. In the Release phase, no new features can be added, nor can any code
+refactoring be planned to be undertaken. Effective communication at this stage
+is crucial to ensure the quality of the release and, consequently, the success.
+
+  * **Alpha**:
+  During this phase, feature development is complete and feedback from the
+  community is being actively addressed. Testing, bug fixing, and documentation
+  updates are ongoing. Any critical issues discovered may lead to the removal of
+  the offending code.
+  * **Beta**:
+  This phase signifies the library is nearing its final release. Testing remains
+  intensive, but the focus is on ensuring stability. While all identified
+  regressions and critical concerns have been addressed, only crucial,
+  last-minute fixes are expected.
+  * **Final**:
+  The Final phase is focused on release preparation. The tasks involved include
+  updating the list of known issues, tagging the code git tree, and issuing
+  digital signatures among others. The phase concludes with the new release
+  becoming publicly available and communicated.
+
+See [Release Requirements Policy] for details on each type of release.
+
+### 5. Support[^3]
+  * **Full Support**:
+  Upon the final release a one-year Full Support is initiated for regular
+  releases, and a four-year period for LTS releases. During this phase, bugs and
+  security issues are addressed and fixed according to the [Stable Release
+  Updates Policy].
+  * **Maintenance Support**:
+  Immediately after the Full Support phase ends, the Maintenance Support phase
+  begins, lasting for one year. During this phase, the primary focus is on
+  fixing security issues, although other bugs may be addressed at the discretion
+  of OpenSSL engineering.
+
+[Stable Release Updates Policy]: https://www.openssl.org/policies/technical/stable-release-updates.html
+[Release Requirements Policy]: https://www.openssl.org/policies/technical/release-requirements.html
+
+[^1]: Feature Acceptance Criteria - 🚧 The document is a work in progress.
+[^2]: Release Steering Committee - A group comprised of four individuals: one
+    internal member from OpenSSL, two specially invited representatives from the
+    community, and the engineering manager. This committee is dedicated to
+    guiding the release cycle, defining release priorities, authorizing release
+    backlog modifications.
+[^3]: We are currently re-evaluating our stable updates release policy. As we
```

Comment:
This footnote seems like the sort of thing that will rapidly get out of date, and not something for the policy itself.

Reply:
True.